### PR TITLE
UPDATE STUDENT .YML

### PR DESCRIPTION
**Pull request by ShivamRai2003**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [ ] Included a Preview link and screenshot showning after and before the changes.
- [x] Images are `240 x 240` [w x h].
- [x] Included a description of change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`

### CODE REVIEW

**Review by nikit19 — APPROVED** [review]

File: _data/students.yml [hunk 1]
```diff
@@ -11,6 +11,14 @@
...
 #   lng: Student's Longitude
 
+
+- name: Shivam Rai  
+  country: India
+  github: ShivamRai2003
+  image: (rai)shivam.jpg 
+  twitter: Shivam Rai
+  
+
```
Comment (nikit19) [inline _data/students.yml]: Looks good just remove this one extra line

Comment (ShivamRai2003) [reply _data/students.yml]: which line

Comment (nikit19) [reply _data/students.yml]: The two blank lines. Remove one of them

**Comment by ParthS007** [comment]
@ShivamRai2003 Please provide a preview link.

**Comment by ShivamRai2003** [comment]
which link

**Comment by ParthS007** [comment]
@ShivamRai2003 Please go through it and read how to generate a preview link https://pages.github.com/

**Review by harshit98 — APPROVED** [review]

**Pull request merged**